README.md

```diff
@@ -206,17 +206,17 @@ Source code layout
 ---
 
 The Redis root directory just contains this README, the Makefile which
-actually calls the real Makefile inside the `src` directory, an example
-configuration for Redis and Sentinel. Finally you can find a few shell
+actually calls the real Makefile inside the `src` directory and an example
+configuration for Redis and Sentinel. Also, you can find a few shell
 scripts that are used in order to execute the Redis, Redis Cluster and
 Redis Sentinel unit tests, which are implemented inside the `tests`
 directory.
 
-Inside the root directory the are the following important directories:
+Inside the root directory following are the important directories:
```

Comment:
`Inside the root directory there are the following important directories:` makes more sense.